test/tools/ossfuzz/yulProto.proto

```diff
@@ -307,6 +310,48 @@ message TerminatingStmt {
 // Stub for a VarDecl without an Expression on the RHS
 message EmptyVarDecl {}
 
+message ForInitStatement {
+  oneof stmt_oneof {
+    AssignmentStatement assignment    = 1;
+    IfStmt              ifstmt        = 2;
+    StoreFunc           storage_func  = 3;
+    Block               blockstmt     = 4;
+    ForStmt             forstmt       = 5;
+    SwitchStmt          switchstmt    = 6;
+    LogFunc             log_func      = 7;
+    CopyFunc            copy_func     = 8;
+    ExtCodeCopy         extcode_copy  = 9;
+    TerminatingStmt     terminatestmt = 10;
+    FunctionCallNoReturnVal funccall  = 11;
+    MultiAssignment     multiassign   = 12;
+  }
+}
+
+message ForInitBlock {
+  repeated ForInitStatement forinit_stmts = 1;
+}
+
+message ForPostStatement {
```

Comment:
Why can't this simply be a Block?

Reply:
Because block includes `break` and `continue` statements

Reply:
But you already skip break and continue if they are illegal in the current context, don't you? Wouldn't it be better to avoid duplication?

Reply:
Ah, you are right! I think the logic should no longer be necessary. Let me recheck and get back on this.

Some notes:
  - We can either not generate of break/continue where they are invalid (via proto spec) or generate them and use logic inside converter
  - I don't see why we should generate them in the first place
  - Therefore, simply having a custom proto spec for "for-init" and "for-post" should suffice.

Reply:
Update on this:

I need the logic to skip break and continue if they are illegal in the current context although the `ForInit` and `ForPost` block proto-spec disallows break/continue because
  - the scope of for-init (or for-post) nested within the scope of an outer for-body is considered scope of for-body
  - we need some mechanism to tell the proto converter that only the inner most scope matters and for this we need to track when we are inside a for-init (or for-post) block

Example of wrong code generation without for-init/post tracking logic

```
{
for {} 1 {}
{
  // Inside for-body
  for {
  // Inside for-init
  break
  } 1 {} {}
}
}
```